lib/rules/scrollable-region-focusable-matches.js

```diff
@@ -54,6 +55,14 @@ function scrollableRegionFocusableMatches(node, virtualNode) {
     }
   }
 
+  /**
+   * if scrollable content is disabled (not focusable)
+   * we can return false
+   */
+  if (attrNames.includes('disabled')) {
```

Comment:
We should use `commons.forms.isDisabled`. It'll cover things like `disabled` being inherited from a parent (such as `fieldset`), and using `aria-disabled`